src/sentry/static/sentry/app/components/charts/worldMapChart.jsx

```diff
@@ -71,13 +86,30 @@ export default class WorldMapChart extends React.Component {
               color: [theme.purpleLightest, theme.purpleDarkest],
             },
             text: ['High', 'Low'],
-            calculable: true,
+
+            // Whether show handles, which can be dragged to adjust "selected range".
+            // False because the handles are pretty ugly
+            calculable: false,
           },
         }}
         {...props}
         yAxis={null}
         xAxis={null}
         series={processedSeries}
+        tooltip={{
+          formatter: ({marker, name, value, seriesName}) => {
+            // If value is NaN, don't show anything because we won't have a country code either
+            if (isNaN(value)) {
+              return '';
+            }
+
+            const formattedValue =
+              typeof value === 'number' ? value.toLocaleString() : value;
+            const countryOrCode = this.state.codeToCountryMap[name] || name;
+
+            return `<div>${marker} ${countryOrCode}: ${formattedValue}</div>`;
```

Comment:
Any chance of XSS here? If `value` isn't a number it is just added to the HTML string.